manual/03-lists/Transforming.md

```diff
@@ -2,3 +2,46 @@ List to list transformations
 ============================
 
 TODO: `map`, `pluck`, `filter`, `aperture`, `uniq`, `zip`, `partition`
+
+Many times we loop over lists for...
+--
+
+**Transforming**
+```js
+const libs = ["Ramda", "Lodash", "Pramda", "Underscore"]
+const html = []
+for(var i = 0; i < libs.length; i++){
+  html[i] = "<li>"+libs[i]+"</li>"
+}
+console.log(html)
+
+```
+Above code is too verbose and we are prone to making mistakes. Ramda provides a `map` function to achieve
```

Comment:
We should link to the functions on first usage:

    ... Ramda provides a [`map`][ma] function to...
    
    [ma]: http://ramdajs.com/docs/#map